index.html

```diff
@@ -211,38 +211,86 @@ <h1>Connect with Fossasia on Social Media</h1>
         </div>
     </div>
 </section>
-
-<div  id="memories" class="bg-secondary">
-    <div >
-        <img class="memories_img" src="img/PastMemories/1.jpg" >
-        <img class="memories_img" src="img/PastMemories/2.jpg" >
-        <img class="memories_img" src="img/PastMemories/3.jpg" >
-        <img class="memories_img" src="img/PastMemories/4.jpg" >
-        <img class="memories_img" src="img/PastMemories/5.jpg" >
+<section>
+    <div class="container">
+    <div class="row">
+    <div class="col-sm-12 text-center">
+    <h4 class="uppercase mb16">Past Memories</h4>
+     <p class="lead mb64">
+For more insight on our events and culture visit <a href="https://www.flickr.com/photos/fossasia/">Flicker</a>
+    </p>
+    </div>
+    </div>
+    
+    <div class="row">
+    <div class="col-sm-12">
+    <div class="lightbox-grid square-thumbs" data-gallery-title="Gallery">
+    <ul>
+    <li>
+    <a href="img/PastMemories/1.jpg" data-lightbox="Gallery">
+    <div class="background-image-holder fadeIn" style="background: url(&quot;img/PastMemories/1.jpg&quot;);">
+    <img alt="image" class="background-image" src="img/PastMemories/1.jpg" style="display: none;">
+    </div>
+    </a>
+    </li>
+    <li>
+    <a href="img/PastMemories/2.jpg" data-lightbox="Gallery">
+    <div class="background-image-holder fadeIn" style="background: url(&quot;img/PastMemories/2.jpg&quot;);">
+    <img alt="image" class="background-image" src="img/PastMemories/2.jpg" style="display: none;">
+    </div>
+    </a>
+    </li>
+    <li>
+    <a href="img/PastMemories/3.jpg" data-lightbox="Gallery">
+    <div class="background-image-holder fadeIn" style="background: url(&quot;img/PastMemories/3.jpg&quot;);">
+    <img alt="image" class="background-image" src="img/PastMemories/3.jpg" style="display: none;">
+    </div>
+    </a>
+    </li>
+    <li>
+    <a href="img/PastMemories/4.jpg" data-lightbox="Gallery">
+    <div class="background-image-holder fadeIn" style="background: url(&quot;img/PastMemories/4.jpg&quot;);">
+    <img alt="image" class="background-image" src="img/PastMemories/4.jpg" style="display: none;">
+    </div>
+    </a>
+    </li>
+    <li>
+    <a href="img/PastMemories/5.jpg" data-lightbox="Gallery">
+    <div class="background-image-holder fadeIn" style="background: url(&quot;img/PastMemories/5.jpg&quot;);">
+    <img alt="image" class="background-image" src="img/PastMemories/5.jpg" style="display: none;">
+    </div>
+    </a>
+    </li>
+    <li>
+    <a href="img/PastMemories/6.jpg" data-lightbox="Gallery">
+    <div class="background-image-holder fadeIn" style="background: url(&quot;img/PastMemories/6.jpg&quot;);">
+    <img alt="image" class="background-image" src="img/PastMemories/6.jpg" style="display: none;">
```

Comment:
Could you please also indent each of the tags properly, as it was before doing changes ?